grunt/tasks/eslint.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var grunt = require('grunt');
+
+module.exports = function() {
+  var done = this.async();
+  grunt.util.spawn({
+    cmd: 'node_modules/.bin/eslint',
+    args: ['src/', 'Gruntfile.js', 'grunt/', 'main.js', 'perf/', 'test/']
```

Comment:
Hmm, can we skip specifying things specifically if we're linting everything? We can blacklist in `.eslintignore`